features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/package.json

```diff
@@ -52,6 +53,7 @@
         "react-dom": "^16.10.2",
         "react-draft-wysiwyg": "^1.13.2",
         "react-dropzone": "^v10.1.7",
+        "react-iframe": "^1.8.0",
```

Comment:
Why we need a library for adding an iframe to react app?

Reply:
Otherwise we have to write a separate component to handle the iframe logic. This library seems to handle all that.

Reply:
removed the react-iframe library